dev_docs/developer_guidelines.md

```diff
@@ -9,6 +9,12 @@
 
 **All instructions are expected to be run from the top level cellxgene directory unless otherwise specified.**
 
+### Environment
+
+For all `make` commands, `common.mk` automatically checks whether required environment variables are set and, if they are not set, assigns them default values from `environment.default`.
+
+You can set these environment variables manually with the `export` shell command, as in `export JEST_ENV=debug`.
+
 ## Running test suite
 Client and server tests run on Travis CI for every push, PR, and commit to master on github. End to end tests run nightly on master only. 
 
```

Comment:
it would be nice to add a note on how to run client unit-tests (for front-end devs)